sound/soc/codecs/rt711-sdca.c

```diff
@@ -809,28 +876,15 @@ static int rt711_sdca_fu0f_event(struct snd_soc_dapm_widget *w,
 	struct snd_soc_component *component =
 		snd_soc_dapm_to_component(w->dapm);
 	struct rt711_sdca_priv *rt711 = snd_soc_component_get_drvdata(component);
-	unsigned char unmute = 0x0, mute = 0x1;
 
 	switch (event) {
 	case SND_SOC_DAPM_POST_PMU:
-		regmap_write(rt711->regmap,
-			SDW_SDCA_CTL(FUNC_NUM_JACK_CODEC, RT711_SDCA_ENT_USER_FU0F,
-				RT711_SDCA_CTL_FU_MUTE, CH_L),
-				unmute);
-		regmap_write(rt711->regmap,
-			SDW_SDCA_CTL(FUNC_NUM_JACK_CODEC, RT711_SDCA_ENT_USER_FU0F,
-				RT711_SDCA_CTL_FU_MUTE, CH_R),
-				unmute);
+		rt711->fu0f_dapm_mute = false;
+		rt711_sdca_set_fu0f_capture_ctl(rt711);
```

Comment:
I am wondering do we really need to mute/unmute it in dapm event? What if we remove the mute/unmute control from damp event? Will there be any side effect if we don't mute/unmute in the event?

Reply:
I

> I am wondering do we really need to mute/unmute it in dapm event? What if we remove the mute/unmute control from damp event? Will there be any side effect if we don't mute/unmute in the event?

I want to put the mute/unmute control at the last operation when capture. So the mixer control just records the status, then the DAPM event decides to mute/unmute.
If DAPM widgets were on, the mixer control will also mute/unmute the control.